projectroles/forms.py

```diff
@@ -541,9 +554,80 @@ def __init__(self, project=None, current_user=None, *args, **kwargs):
             self.fields['description'].widget = forms.HiddenInput()
             self.fields['readme'].widget = forms.HiddenInput()
 
+    @classmethod
+    def _app_settings_validation(
```

Comment:
It is usually convention to name functions/methods as a verb, so something like `_validate_app_settings()` would be more appropriate.